### DISCUSSION TITLE
A320 Neo Issues

### DISCUSSION THREAD

**Discussion by dashawn11** [flybywiresim/aircraft#3859] [Help] [answered]
I just updated to latest stable version of A320 Neo. My plane now taxis violently to right and takes off violently to the right. I tried uninstalling and reinstalling the plane and it still does the same. Please help. I am trying to do my first flight.

**Comment by ExampleWasTaken** [comment, marked as answer]
Should you use a joystick or a yoke, try increasing your deadzones to about 25%. If that resolves the issue you can slowly decrease them until you start to notice the issue happening again.
